extras/i2c/i2c.h

```diff
@@ -32,26 +32,71 @@
 extern "C" {
 #endif
 
-// Init bitbanging I2C driver on given pins
+// I2C driver for ESP8266 written for use with esp-open-rtos
+// Based on https://en.wikipedia.org/wiki/I²C#Example_of_bit-banging_the_I.C2.B2C_Master_protocol
+// With calling overhead, we end up at ~320kbit/s
+
+///////Level 0 API
```

Comment:
It's probably better to stick with a uniform comment style everywhere, i.e. `//`